tests/deriver/conftest.py

```
@@ -0,0 +1,309 @@
+            task_type=task_type,
+        )
+        db_session.add(active_session)
+        await db_session.commit()
+        await db_session.refresh(active_session)
+        return active_session
+
+    return _create_active_session
+
+
+@pytest.fixture
+def mock_queue_manager(monkeypatch: pytest.MonkeyPatch) -> AsyncMock:  # pyright: ignore[reportUnusedParameter]
```

Ruff (0.12.2)

Unused function argument: `monkeypatch`

(ARG001)